policyengine_us/variables/gov/states/hi/tax/income/credits/cdcc/dependent_care_benefit/hi_min_head_spouse_earned.py

```diff
@@ -0,0 +1,64 @@
+from policyengine_us.model_api import *
+
+
+class hi_min_head_spouse_earned(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Hawaii minimum income between head and spouse"
+    defined_for = StateCode.HI
+    unit = USD
+    definition_period = YEAR
+    reference = "https://files.hawaii.gov/tax/forms/2022/n11ins.pdf#page=28"
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.hi.tax.income.credits.cdcc
+        person = tax_unit.members
+        head = person("is_tax_unit_head", period)
+        spouse = person("is_tax_unit_spouse", period)
+        # earned minimum income if is disabled/full-time student
+        head_eligible = head & (
```

Comment:
```suggestion
        head_eligible = head_or_spouse & (
```

Reply:
this will give us its own vectorized output for each head and spouse - we can consolidate the calculation if the steps are the same for head or spouse